feeds.conf.default

```diff
@@ -5,5 +5,6 @@ src-git telephony https://github.com/openwrt/telephony.git
 src-git management https://github.com/openwrt-management/packages.git
 src-git targets https://github.com/openwrt/targets.git
 src-git ci40 https://github.com/CreatorDev/Ci40-platform-feed.git
+src-git caskeid https://imgworks.imgtec.com:8082/systems/openwrt-caskeid-feeds.git
```

Comment:
Is it publicly accessibile by anyone?  How arr you managing the credentials ? netrc way?